app/src/server/trpc/routers/service.ts

```diff
@@ -37,7 +37,11 @@ export const serviceRouter = router({
         resArtifacts => resArtifacts.json() as Promise<fhir4.Bundle<FhirArtifact>>
       );
       const artifactList = artifactBundle.entry?.map(entry => ({
-        label: entry.resource?.name || entry.resource?.id || '',
+        label:
+          entry.resource?.name?.concat(`|${entry.resource?.version}`) ||
```

Comment:
```suggestion
          entry.resource?.name?.concat(`|${entry.resource.version}`) ||
```
don't need the second resource `?` because it's already been checked in this line